README.md

```diff
@@ -8,40 +8,83 @@ Application to be used with the [EVAL-CN0363-PMDZ](https://wiki.analog.com/resou
 
 
 ## Build Instructions
-
-``` 
+### Clone the repository
+```
 $ git clone https://github.com/analogdevicesinc/colorimeter.git
 Cloning into 'colorimeter'...
 remote: Counting objects: 65, done.
 remote: Compressing objects: 100% (33/33), done.
 remote: Total 65 (delta 13), reused 2 (delta 2), pack-reused 28
 Unpacking objects: 100% (65/65), done.
 Checking connectivity... done.
+$ cd colorimeter/
+```
+### Install dependencies:
+```
+$ sudo apt install $(cat requirements.txt) -y
+```
+
+### Build app
```

Comment:
Some of the subtitles have an empty line before, and some of them not. Please uniformize them.
I would go with an empty before each titles, inclusive between "Build Instructions" and "Clone the repo"